.env.development

```diff
@@ -9,6 +9,6 @@ CHECKOUT_POLLING_ACTIVATION_ATTEMPTS=3
 CHECKOUT_ENV=develop
 CHECKOUT_RECAPTCHA_SITE_KEY=6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI
 CHECKOUT_DONATIONS_URL=https://api.uat.platform.pagopa.it/donations/api/v1/availabledonations
-CHECKOUT_ECOMMERCE_HOST=http://localhost:1234
+CHECKOUT_ECOMMERCE_HOST=http://localhost:8080
```

Comment:
Doesn't this clash with CORS?

Reply:
Need to revert it. Use it for local testing purpose.

Reply:
Anyway this is .env.development